hypertrace-ingester/build.gradle.kts

```diff
@@ -27,6 +27,7 @@ hypertraceDocker {
 dependencies {
 
   implementation(libs.hypertrace.kafkaStreams.framework)
+  implementation(platform(libs.hypertrace.kafka.bom))
```

Comment:
bom is already part of kafkastreams framework, don't need to explicitly declaring it when using the framework.